config/routes.rb

```diff
@@ -3,22 +3,19 @@
   	controllers: { omniauth_callbacks: 'users/omniauth_callbacks' }
```

Comment:
that' so clean now!
We need to discuss cases where you're supposed (or not) to use 'resources' in routes.